### DISCUSSION TITLE
Feature Request: Window Names & Bug: New macro command opens in background

### DISCUSSION THREAD

**Discussion by StuartGauffi** [audacity/audacity#5663] [Ideas & Feature Requests]
**Bug:**
While experimenting with 3.4.2 to try to find a workaround for Export Selected Audio, I discovered that the New macro command creates a new project in the background, where Command-N and File | New create the new project in the foreground.  This needs to be fixed.

But that also revealed that all Audacity windows are named "Audacity" until the project is saved.  This makes it impossible to identify the window using a system-level macro tool.

**Feature Request:**
- Name the window for an unsaved Audacity project "Audacity - unnamed". Additional unsaved projects should be named "Audacity - unnamed[1]", "Audacity - unnamed[2]", etc.
- Name the window for a saved Audacity project "Audacity - Whatever Project Name I Chose"

**Comment by petersampsonaudacity** [comment]
@StuartGauffi 

I created an Issue for this ENH to give it better visiblity.

See #5696
